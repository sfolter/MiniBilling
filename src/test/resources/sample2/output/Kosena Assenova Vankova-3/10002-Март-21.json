{
    "documentDate": "2022-08-10T16:37:26.990156600",
    "documentNumber": "10002",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": 335.72,
    "totalAmountWithVat": 402.864,
    "lines": [
        {
            "index": 1,
            "quantity": 392.4,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.7,
            "priceList": 1,
            "amount": 274.68
        },
        {
            "index": 2,
            "quantity": 43.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 61.04
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": [
                1,
                2
            ],
            "percentage": 20,
            "amount": 54.936
        },
        {
            "index": 2,
            "lines": [
                1,
                2
            ],
            "percentage": 20,
            "amount": 12.208
        }
    ]
}