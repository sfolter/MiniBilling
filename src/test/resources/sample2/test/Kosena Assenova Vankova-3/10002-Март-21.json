{
    "documentDate": "2022-08-10T11:09:25.082105100",
    "documentNumber": "10002",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": 335.72,
    "lines": [
        {
            "index": 1,
            "quantity": 392.4,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.7,
            "priceList": 2,
            "amount": 274.68
        },
        {
            "index": 2,
            "quantity": 43.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 2,
            "amount": 61.04
        }
    ]
}