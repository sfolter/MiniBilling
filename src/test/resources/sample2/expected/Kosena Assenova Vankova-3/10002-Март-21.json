{
  "documentNumber": "10002",
  "consumer": "Kosena Assenova Vankova",
  "reference": "3",
  "totalAmount": 350.07,
  "lines": [
    {
      "index": 1,
      "quantity": 371.908,
      "lineStart": "2021-01-01T14:40:00Z",
      "lineEnd": "2021-02-28T21:59:59Z",
      "product": "gas",
      "price": 0.7,
      "priceList": 1,
      "amount": 260.34
    },
    {
      "index": 2,
      "quantity": 64.092,
      "lineStart": "2021-02-28T22:00:00Z",
      "lineEnd": "2021-03-11T07:00:00Z",
      "product": "gas",
      "price": 1.4,
      "priceList": 1,
      "amount": 89.73
    }
  ]
}